test/examples_test.go

```diff
@@ -15,25 +15,45 @@ func TestCoinPricesFromCoingecko(t *testing.T) {
 		"YOUR_COINGECKO_API_KEY",
 	}
 
-	NewProjectTest(t, projectDir).
+	NewTestscriptTest(t, projectDir).
 		ExecuteMakeTarget("build").
 		CopyFile("tmp/x.wasm").
 		CopyFile("config.toml").
 		RenderTemplateFileFromEnvWithCleanup("fn-call.json", requiredEnvVars).
-		RunScript(filepath.Join(scriptDir, projectName+".txtar"))
+		Run(filepath.Join(scriptDir, projectName+".txtar"))
 }
 
 func TestErrorHandlingAttestFnCall(t *testing.T) {
 	projectName := "error_handling_attest_fn_call"
 	projectDir := filepath.Join(examplesDir, projectName)
-	NewProjectTest(t, projectDir).
+	NewTestscriptTest(t, projectDir).
 		ExecuteMakeTarget("build").
 		CopyFile("tmp/x.wasm").
 		CopyFile("config.toml").
 		CopyFile("successFunc.json").
 		CopyFile("errorFunc.json").
 		CopyFile("panicFunc.json").
-		RunScript(filepath.Join(scriptDir, projectName+".txtar"))
+		Run(filepath.Join(scriptDir, projectName+".txtar"))
+}
+
+func TestErrorHandlingOnChain(t *testing.T) {
```

Comment:
Went ahead and added on-chain tests for hello-world, error handling, and twap even though the card just called for hello-world. It wasn't hard and I thought it would be a better demonstration of how `HardhadTest` works to have multiple examples

Reply:
Great, agreed it's easier to set up test tooling/harness with a few examples